wolfcrypt/src/ecc.c

```diff
@@ -9073,10 +9088,8 @@ static int accel_fp_mul2add(int idx1, int idx2,
                                                               mp)) != MP_OKAY) {
                 break;
              }
-          }
 
-          /* add if not first, otherwise copy */
-          if (!first) {
+             /* add if not first, otherwise copy */
```

Comment:
Why is !first check removed here?

Reply:
Merging the two blocks as they both had the same if (!first) check on them.
cppcheck complained about redundancy in these checks.